ec-europa/europa-component-library
# refactor(images): make component extendable - INNO-677

# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:


Not relevant:
